book/src/external-provisioner.md

```diff
@@ -8,6 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version
 --|--|--|--|--|--|--|--
+[external-provisioner v1.1.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.1.1) | [release-1.1](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.1.1 | v1.13 | - | v1.14
 [external-provisioner v1.1.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.1.0) | [release-1.1](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.1.0 | v1.13 | - | v1.14
```

Comment:
let's remove the line for 1.1.0 so that we only document the latest release in a minor version

Reply:
Updated to remove 1.1.0